### PR TITLE
SPM Support

### PR DESCRIPTION
**Pull request by BalazsSzamody**
- Package manifest added
- UIKit condition added
- .gitignore updated to ignore SPM generated files
- PodSpec version increased

### CODE REVIEW

File: Sources/MSAutoView/MSAutoView.swift [hunk 1]
```diff
@@ -0,0 +1,3 @@
+struct MSAutoView {
+    var text = "Hello, World!"
+}
```
Comment (MaherKSantina) [inline Sources/MSAutoView/MSAutoView.swift]: It looks like this is dummy code, should it be removed?

File: Tests/MSAutoViewTests/MSAutoViewTests.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import XCTest
+@testable import MSAutoView
+
+final class MSAutoViewTests: XCTestCase {
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct
+        // results.
+        XCTAssertEqual(MSAutoView().text, "Hello, World!")
```
Comment (MaherKSantina) [inline Tests/MSAutoViewTests/MSAutoViewTests.swift]: It looks like this is dummy code, should it be removed?

**Comment by BalazsSzamody** [comment]
I modified the minimum target to iOS 8.0.
Removed the unused content generated by SwiftPM during package init.

**Comment by MaherKSantina** [comment]
Awesome! 👏  Is there a way I can test this to make sure that it's working properly?

**Comment by BalazsSzamody** [comment]
You can try to import it into another Package.

Or you can try to import it in a project if you have an Xcode 11 beta

**Comment by BalazsSzamody** [comment]
I think you can test this easily now. ;)

**Pull request merged**